recursio-client/Global/Server.gd

```diff
@@ -30,7 +30,7 @@ signal game_result(winning_player_id)
 signal player_hit(hit_player_id, perpetrator_player_id, perpetrator_timeline_index)
 signal ghost_hit(hit_ghost_player_owner, hit_ghost_id, perpetrator_player_id, perpetrator_timeline_index)
 signal quiet_ghost_hit(hit_ghost_player_owner, hit_ghost_id, perpetrator_player_id, perpetrator_timeline_index)
-signal timeline_picks(player_pick, enemy_picks)
+signal timeline_pick(picking_player_id, timeline_index)
```

Comment:
The old signal had the same problem, but it's better imo to use the passive form for signal names, so "timeline_picked" in this case. Makes it a bit clearer that something just happened and can be reacted to, rather than something which needs to be done.

Reply:
Fixed